avocado/utils/memory.py

```diff
@@ -41,30 +41,16 @@ def read_from_meminfo(key):
     return int(re.search(r'\d+', meminfo).group(0))
 
 
-def memtotal():
-    """
-    Read ``Memtotal`` from meminfo.
-    """
-    return read_from_meminfo('MemTotal')
-
-
-def freememtotal():
-    """
-    Read ``MemFree`` from meminfo.
-    """
-    return read_from_meminfo('MemFree')
-
-
```

Comment:
They might be superfluous and simplistic, but the fact is that we have users relying on them:
```
~/git/avocado-misc-tests (master)$ fgrep -r memtotal *
io/disk/disktest.py:        memory_mb = memory.memtotal() / 1024
memory/eatmemory.py:        mem = self.params.get('memory_to_test', default=memory.memtotal())
memory/memtester.py:            memory.freememtotal() / 1024))
memory/stutter.py:        mem_byte = str(memory.memtotal())
memory/transparent_hugepages.py:        free_mem = int(memory.freememtotal() / 1024)
memory/transparent_hugepages_defrag.py:        total = memory.memtotal()
memory/memcached.py:                                        default=(memory.freememtotal() / 1024))
perf/stress.py:            mb = (memory.freememtotal() +
```
Who knows who else is using them... Please do your thing in `rounded_memtotal`, but don't touch it here.